modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -474,6 +479,76 @@ public int getOneToManyRelatedObjectEntriesCount(
 		return objectEntryPersistence.dslQueryCount(dslQuery);
 	}
 
+	@Override
```

Comment:
@marcelabc It's kind of out of pattern returning a `List<BaseModel<?>>` in one of the ObjectEntryLocalServiceImpl methods. Maybe we can put this inside `SystemObject1toMObjectRelatedModelsProviderImpl`. What do you think?

Reply:
btw, I think we'll have also a different implementation for this when we have MtoM relationship. Am I right?